tests/suite/cli-ui/rustup/rustup_help_cmd_stdout.toml

```diff
@@ -9,11 +9,11 @@ The Rust toolchain installer
 Usage: rustup[EXE] [OPTIONS] [+toolchain] [COMMAND]
 
 Commands:
+  toolchain    Install, uninstall, or list toolchains
   show         Show the active and installed toolchains or profiles
   update       Update Rust toolchains and rustup
   check        Check for updates to Rust toolchains and rustup
   default      Set the default toolchain
```

Comment:
Looking at the diff it makes me wonder if `default`/`target`/`component` should be moved to the top as well since they basically make up the main concerns of this project. It'd be fine just to put them under `toolchain` I guess.

Reply:
I thought about that -- I could see your point for `default` but I think `target` and `component` are probably much less likely to be used by many users, especially inexperienced users just getting started?

Like, I think the vast majority of users probably installs a default toolchain and then updates it over time, and doesn't do much else.

Reply:
> I thought about that -- I could see your point for `default` but I think `target` and `component` are probably much less likely to be used by many users, especially inexperienced users just getting started?
> 
> Like, I think the vast majority of users probably installs a default toolchain and then updates it over time, and doesn't do much else.

Thanks for the quick reply! I think it will make sense as well to just move `default` like you've pointed out. After all, this is just my additional remark, I don't want it to be a blocker :)